lib/widgets/lightbox.dart

```diff
@@ -256,10 +371,13 @@ class _ImageLightboxPageState extends State<_ImageLightboxPage> {
     return BottomAppBar(
       color: color,
       elevation: elevation,
-      child: Row(children: [
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
Is there a reference design for this? Either way, it would be helpful to post screenshots to [#mobile](https://chat.zulip.org/#narrow/channel/48-mobile) and see if it's the desirable design to go with.